netlify/edge-functions/dev-card.tsx

```diff
@@ -0,0 +1,381 @@
+import React from "react";
+import { ImageResponse } from "og_edge";
+import type { Config } from "https://edge.netlify.com";
+import { getLocalAsset } from "../og-image-utils.ts";
+
+const MAX_ABOUT_LENGTH = 77;
+const baseApiUrl = Deno.env.get("NEXT_PUBLIC_API_URL");
+
+function getTopPercent(oscr: number) {
+  switch (true) {
+    case oscr >= 250:
+      return 1;
+    case oscr >= 235:
+      return 2;
+    case oscr >= 225:
+      return 3;
+    case oscr >= 215:
+      return 4;
+    case oscr >= 200:
+      return 5;
+    default:
+      return "";
+  }
+}
+function differenceInDays(dateLeft, dateRight) {
+  // Convert both dates to milliseconds
+  const dateLeftMs = dateLeft.getTime();
+  const dateRightMs = dateRight.getTime();
+
+  // Calculate the difference in milliseconds
+  const differenceMs = Math.abs(dateLeftMs - dateRightMs);
+
+  // Convert the difference to days
+  const days = Math.floor(differenceMs / (1000 * 60 * 60 * 24));
+
+  return days;
+}
+
+const getContributorPullRequestVelocity = (repositoryPullRequests: any[]) => {
+  const mergedPRs = repositoryPullRequests.filter((prState) => prState.pr_is_merged);
+
+  const totalDays = mergedPRs.reduce((total, event) => {
+    const daysBetween = differenceInDays(new Date(event.pr_closed_at), new Date(event.pr_created_at));
+    return (total += daysBetween);
+  }, 0);
+
+  const averageVelocity = mergedPRs.length > 0 ? totalDays / mergedPRs.length : undefined;
+
+  if (averageVelocity && averageVelocity < 1) {
+    return 1;
+  }
+
+  return averageVelocity ? Math.floor(averageVelocity) : averageVelocity;
+};
+
+const getPullRequestsHistogramToDays = (pull_requests: any[], range = 30) => {
+  const graphDays = pull_requests.reduce((days: { [name: string]: number }, curr: any) => {
+    const day = differenceInDays(new Date(), new Date(curr.bucket));
+
+    if (days[day]) {
+      days[day] += curr.prs_count;
+    } else {
+      days[day] = curr.prs_count;
+    }
+
+    return days;
+  }, {});
+
+  const days: any[] = [];
+
+  for (let d = range; d >= 0; d--) {
+    days.push({ x: d, y: graphDays[d] || 0 });
+  }
+
+  return days;
+};
+
+const CrownIcon = () => {
```

Comment:
Had to copy the SVG for these icon components in the Next.js codebase for the same reason above.